docs/attributes-registry/process.md

```diff
@@ -11,12 +11,27 @@
 | `process.command_line` | string | The full command used to launch the process as a single string representing the full command. On Windows, can be set to the result of `GetCommandLineW`. Do not set this if you have to assemble it just for monitoring; use `process.command_args` instead. | `C:\cmd\otecol --config="my directory\config.yaml"` |
+| `process.end` | string | The date and time the process ended, in ISO 8601 format. | `2023-11-21T09:26:12.315Z` |
 | `process.executable.name` | string | The name of the process executable. On Linux based systems, can be set to the `Name` in `proc/[pid]/status`. On Windows, can be set to the base name of `GetProcessImageFileNameW`. | `otelcol` |
 | `process.executable.path` | string | The full path to the process executable. On Linux based systems, can be set to the target of `proc/[pid]/exe`. On Windows, can be set to the result of `GetProcessImageFileNameW`. | `/usr/bin/cmd/otelcol` |
+| `process.exit_code` | int | The exit code of the process. | `127` |
+| `process.group_leader.pid` | int | The PID of the process's group leader. This is also the process group ID (PGID) of the process. | `23` |
+| `process.interactive` | boolean | Whether the process is connected to an interactive shell. |  |
 | `process.owner` | string | The username of the user that owns the process. | `root` |
 | `process.parent_pid` | int | Parent Process identifier (PPID). | `111` |
 | `process.pid` | int | Process identifier (PID). | `1234` |
+| `process.real_user.id` | int | The real user ID (RUID) of the process. | `1000` |
+| `process.real_user.name` | string | The username of the real user of the process. | `operator` |
 | `process.runtime.description` | string | An additional description about the runtime of the process, for example a specific vendor customization of the runtime environment. | `Eclipse OpenJ9 Eclipse OpenJ9 VM openj9-0.21.0` |
 | `process.runtime.name` | string | The name of the runtime of this process. For compiled native binaries, this SHOULD be the name of the compiler. | `OpenJDK Runtime Environment` |
 | `process.runtime.version` | string | The version of the runtime of this process, as returned by the runtime without modification. | `14.0.2` |
+| `process.saved_user.id` | int | The saved user ID (SUID) of the process. | `1002` |
+| `process.saved_user.name` | string | The username of the saved user. | `operator` |
+| `process.session_leader.pid` | int | The PID of the process's session leader. This is also the session ID (SID) of the process. | `14` |
+| `process.start` | string | The date and time the process started, in ISO 8601 format. | `2023-11-21T09:25:34.853Z` |
+| `process.user.id` | int | The effective user ID (EUID) of the process. | `1001` |
```

Comment:
how this would be related to https://github.com/open-telemetry/semantic-conventions/pull/731?

Reply:
this will/might be changed later to use `user` namespace as soon as it's available and we have implemented re-use functionality.

Reply:
do I understand correctly, that the markdown won't change and we'd have `process.user.id` attribute?
Why not `user.id` - do we need more than one user?

Reply:
> Why not user.id - do we need more than one user?

(Assuming we come to a conclusion on https://github.com/open-telemetry/semantic-conventions/pull/731#discussion_r1511480517)
A signal could have different "users" at the same time. For example a `process.user.id` is the OS user that is running the process and in addition there could be a `client.user.id` (this is a field in ECS, not present in SemConv, yet) describing the "enduser" that authenticated through the client. So having only `user.*` without context is ambiguous.

Reply:
Leaving aside the mechanics of how inclusion/embedding/reuse can be done, what I hear that the final goal is to have `process.user.id` because `user.id` will be confusing because of all the other potential users associated with the same telemetry item. Makes sense!